### DISCUSSION TITLE
Workflows

### DISCUSSION THREAD

**Discussion by njbowen** [biostars/biostar-handbook-issues#205] [Q&A] [answered]
General Inquiry; not time-sensitive.  Exciting new addition!  Thanks.  I'm just beginning to look over Workflows and this question may be premature as I realize this is a work in progress.  When I'm navigating Workflows I don't see a column for the Workflow web page to the left, that is, what is labeled as "pages" in the "Navigating content" section of Workflows.  Is that coming later as in the example page that is shown?  I'm using Chrome latest on Mac OS 12.3.1.

**Comment by ialbert** [comment, marked as answer]
because of the re-writing editing, the exact naming of the pages may change, 

perhaps providing a screenshot there was a bit premature, the primary purpose was just to indicate the location of  navigation links

I may blur out the contents there so that not to be out of sync
